src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
I suggest exporting the API URL only in one place in your application, e.g. if you decide to change it, you will need to change it only once.